src/main/java/org/broadinstitute/hellbender/utils/test/ArgumentsBuilder.java

```diff
@@ -93,6 +93,36 @@ public ArgumentsBuilder addFileArgument(String argumentName, File file){
     }
 
     /**
+     * add an argument with a file as its parameter
+     */
+    public ArgumentsBuilder addBooleanArgument(String argumentName, boolean yes){
+        Utils.nonNull(argumentName);
+        add("--" + argumentName);
+        add(yes);
+        return this;
+    }
+
+    /**
+     * add an argument with a given value to this builder
+     */
+    public ArgumentsBuilder addArgument(final String argumentName, final String argumentValue) {
+        Utils.nonNull(argumentValue);
+        Utils.nonNull(argumentName);
+        add("--" + argumentName);
+        add(argumentValue);
+        return this;
+    }
+
+    /**
+     * add a positional argument to this builder
+     */
+    public ArgumentsBuilder addPositionalArgument(final String argumentName) {
```

Comment:
I think positional arguments may have to go at the end... (I'm not certain, I should test it, we might be smart enough to detect them in the middle.)  Could you check if it works out of order, and if it doesn't make it save the positional ones separately to add at the end?